JSDemos/Demos/Scheduler/Resources/description.md

```diff
@@ -1,3 +1,23 @@
 
-[0]: /Documentation/ApiReference/UI_Components/dxScheduler/Configuration/resources/#useColorAsDefault
\ No newline at end of file
+To add a resource, do the following:
+
+### Add resource instances
+Resource instances are objects that define the available categories. Each resource instance object must contain the id, text, and color fields. Create an array of such objects. If your field names differ, specify them in the [valueExpr](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/resources/valueExpr.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/resources/#valueExpr'), [colorExpr](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/resources/colorExpr.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/resources/#colorExpr') and [displayExpr](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/resources/displayExpr.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/resources/#displayExpr') properties.
+
+### Define a resource kind object in the [resources](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/resources '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/resources/') property
+The resource property is an array that stores resource kind objects. Each object should have at least the following fields:
+
+- [dataSource](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/resources/dataSource.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/resources/#dataSource')         
+    Maps a resource kind to resource instances. For details on how to provide data, see the [Data Binding](/Documentation/Guide/Data_Binding/Specify_a_Data_Source/Local_Array/) help topics.
+- [fieldExpr](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/resources/fieldExpr.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/resources/#fieldExpr')             
+    Specifies the name of an appointment data field that assigns the appointment to a specific category id.
+
+Create one resource kind object. To enable multiple category selection, set the object's [allowMultiple](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/resources/#allowMultiple) property to **true**. 
+
+By now, users should be able to select categories in the newly created appointment form field.
+
+### Optionally, preset appointments to specific categories in code
+To associate a specific appointment with the resource, include a field specified in the **fieldExpr** property into the appointment's data object. This field should be equal to one of the appointment instances' id.
```

Comment:
We can preset something only in code, right? I also don't think that the word optionally is a good choice for a heading. The last point - Ray doesn't like apostrophes after plurals.

```suggestion
### Preset appointments to specific categories
To associate a specific appointment with the resource, include a field specified in the **fieldExpr** property into the appointment's data object. This field's should be equal to id of one of the appointment instances.
```